dune

```diff
@@ -1,5 +1,10 @@
 (env
-  (_
-    (flags (:standard -w @a-4-6-20-29-35-41-42-44-45-48-50 -error-style short ))
+  (dev
+    (flags (:standard  -w -6-50 -error-style short ))
```

Comment:
I'd suggest keeping this in sync with _tags, otherwise we're going to keep breaking the dune build inadvertently.

Reply:
sounds good